packages/api/src/controllers/project.ts

```diff
@@ -0,0 +1,163 @@
+import { Request, RequestHandler, Router } from "express";
+import { authorizer, validatePost } from "../middleware";
+import { db } from "../store";
+import { v4 as uuid } from "uuid";
+import {
+  makeNextHREF,
+  parseFilters,
+  parseOrder,
+  getS3PresignedUrl,
+  toStringValues,
+  pathJoin,
+  getObjectStoreS3Config,
+  reqUseReplica,
+  isValidBase64,
+  mapInputCreatorId,
+} from "./helpers";
+import sql from "sql-template-strings";
+import {
+  ForbiddenError,
+  UnprocessableEntityError,
+  NotFoundError,
+  BadRequestError,
+  InternalServerError,
+  UnauthorizedError,
+  NotImplementedError,
+} from "../store/errors";
+
+const app = Router();
+
+export async function getProject(req, projectId) {
```

Comment:
Remember to add types. TypeScript will also help you get around!
```suggestion
export async function getProject(req: Request, projectId: string) {
```

Reply:
Done